### PR TITLE
Twitter api

### PR DESCRIPTION
**Pull request by michaelko92**

### CODE REVIEW

**Review by sveranicola — APPROVED** [review]
Looks Great !! 🛸

**Review by bakese — APPROVED** [review]
Looks good

File: API/twitterAPI.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+var T = new Twit(config);
+
+var params = {
+  count: 20,
+  exclude_replies: true,
+  tweet_mode:'extended'
+}
+
+var formatRetrievedData = (err, data, respons) => {
```
Comment (gregorymacat) [inline API/twitterAPI.js:13]: do you ever use the response paramter here? Also it has a typo.

File: API/twitterAPI.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+  console.log(tweets);
+};
+
+getTweet = () => {
+  T.get('statuses/home_timeline', params, formatRetrievedData);
+}
+
+postTweet = () => {
+  T.post('statuses/update', { status: 'hello world!' }, function(err, data, response) {
```
Comment (gregorymacat) [inline API/twitterAPI.js:52]: What is the difference between data and response in this callback function here?

**Review by gregorymacat — APPROVED** [review]
Left some comments to consider for later

**Pull request merged**